docs/en/docs/configurations/template.md

```diff
@@ -20,7 +20,7 @@ You are free to build your own and pass it to the `TemplateConfig`. This way you
     You will notice the name of the parameters in the `TemplateConfig` match maority of the jinja2 implementation.
 
 !!! Warning
-    The `Mako` engine has a limited integration within Esmerald. This will change in the future.
+    The `Mako` engine integration is deprecated.
```

Comment:
The `Mako` engine integration, due to its own limitations will be not supported by Esmerald from version 3.6.0.